web-common/src/components/tooltip/StackingWord.svelte

```diff
@@ -32,7 +32,5 @@
     @apply inline-block;
     border-radius: 2px;
     position: relative;
-    mix-blend-mode: screen;
-    background-blend-mode: screen;
```

Comment:
Not sure what this does but the Copy animation in light mode looks weird and has a momentarily dark bg